### ISSUE TITLE
The UI is broken and doesn't adjust to fit the window

### ISSUE THREAD

**Issue by Katzenwerfer** [williamckha/spicetify-fluent#66]
### 🔍 Is there already an issue for your problem?

- [X] I have checked older issues, open and closed

### ℹ Environment / Computer Info

```Markdown
- Spotify version: Spotify for Windows 1.1.83.956.gbe7ea9a7
- Spicetify version: 2.9.6
- Fluent installation method: powershell or marketplace
```

### 📝 Description

For the most part, the interface doesn't display the sidebar's icons correctly and certain items in the dark color scheme are barely readable or transparent.

It also has an issue with the window scale, which causes it to not fit the screen correctly.

### 📸 Screenshots

![image](https://user-images.githubusercontent.com/53267697/164762480-55133a5a-c5dc-420e-89d7-743019958fb0.png)

**Comment by rlaphoenix** [comment]
You did not correctly list the installation method, which one is it? Spotify versions up to 1.81.x is confirmed to be supported with 1.82.x having reports of incompatibilities. It would seem 1.82.x has some problems, whereas 1.83.x has major problems. Have you tried downgrading to 1.81.x for now?

**Comment by Katzenwerfer** [comment]
> You did not correctly list the installation method, which one is it?
I have tried both and the issue persists

I haven't tried downgrading, but if that's the only possible fix, then I'll just wait for any potential fixes.

**Comment by mirronake** [comment]
Same

**Comment by skizv** [comment]
samee

**Comment by Katzenwerfer** [comment]
By the way, I can gather more details of it, like logs and other stuff, if that can be useful.

**Comment by nigelonius** [comment]
![image](https://user-images.githubusercontent.com/87260000/165031342-642f88bc-c329-4bca-bdbf-56ae261e0609.png)
I am having the same probem with ver 1.83

**Comment by senolfndk** [comment]
same problem ![Ekran görüntüsü 2022-04-26 224547](https://user-images.githubusercontent.com/52818836/165380102-cfb3a0ba-adc6-4c32-b260-21a662b7b409.png)

**Comment by nigelonius** [comment]
IS this issue being worked on? It's been almost a week now...asking for a friend 🙃

**Comment by nanchaara** [comment]
ikr, its just getting worse 
i think fluent is just no longer developed (?)
![image](https://user-images.githubusercontent.com/71387731/165684913-73245571-94f3-40b8-a8e3-d112833bb342.png)

**Comment by HuyHung1408** [comment]
Same! Hope that there will be a fix soon

**Comment by JonnyBoy2000** [comment]
Yeah I have the same issue. The UI is all buggy but I love the theme so I hope a fix comes out.
![image](https://user-images.githubusercontent.com/22266893/166868372-59b1ea4f-b59f-4441-8ba9-dadf770ddea8.png)
![image](https://user-images.githubusercontent.com/22266893/166868354-4cb1c8a5-0038-4387-b047-a83a8482e922.png)
![image](https://user-images.githubusercontent.com/22266893/166868344-13a3c5e7-40dd-4393-a7c2-4de4a12fb13e.png)

**Comment by rodolfol95** [comment]
Same here still broken, I don't get it why Spotify its on and endless fight to avoid their app getting themed?

**Comment by rodolfol95** [comment]
Opened a ticket on Spicetify and they closed it saying it wasn't on their end but didnt say what was causing it? so what's going on? all themes are broken on latest spotify version

**Comment by rlaphoenix** [comment]
We ultimately need to wait for Spicetify-cli to get official and full support for newer Spotify versions. Spotify seems to have been pumping out new "major" versions faster and faster lately. All likely to be to prevent theming.

**Comment by ShadowPlayzzYT** [comment]
I'm actually thinking about fixing it myself and release it on my profile. Should i try that?

**Comment by rlaphoenix** [comment]
> I'm actually thinking about fixing it myself and release it on my profile. Should i try that?

If you would like to contribute, it would be best to make a fork, apply your changes to a new branch, then make a pull request from your new branch to this repo's master.

That way we can review the changes and pull it for everyone to use. Fixing it yourself might be possible but issues caused by spicetify-cli not having full support for the latest versions might be problematic.

**Comment by ShadowPlayzzYT** [comment]
I could do that, the thing is just that I don't have much time cause the german school system fucks up but i'll try anyway

**Comment by Paxxous** [comment]
Having the same error myself.

**Comment by Pyroks** [comment]
![image](https://user-images.githubusercontent.com/91220105/199228438-ec2c9715-367d-44f7-9c2f-de3420f223df.png)
